<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Weaver Application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(201, 209, 211);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item row="3" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>150</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="steelCathodicButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(70, 88, 129);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Steel Cathodic Protection
Measurement</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QPushButton" name="ironCathodicButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(70, 88, 129);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Iron Cathodic Protection
Measurement</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="TitleLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Pipe Measurement Application&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="ironThicknessButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(70, 88, 129);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Iron Wall Thickness
Measurement</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="steelThicknessButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(70, 88, 129);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Steel Wall Thickness
Measurement</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="ExplanationLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>1280</width>
          <height>720</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Welcome to Pipe Measurement Application!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Selamat datang di Applikasi Pipe Measurement!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Silahkan pilih menu perhitungan yang diinginkan, kemudian masukan input sesuai data yang telah diambil.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Tekan Home untuk kembali ke layar awal untuk memulai kembali!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="IronLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Iron Measurement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="SteelLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Steel Measurement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
